quality_control_mrp_oca/views/mrp_production_view.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!-- Copyright 2018 Simone Rubino - Agile Business Group
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="action_qc_inspection_per_production" model="ir.actions.act_window">
+        <field name="name">Quality inspections from production order</field>
+        <field name="res_model">qc.inspection</field>
+        <field name="view_mode">tree,form</field>
+        <field name="domain">[('production_id', '=', active_id)]</field>
```

Comment:
here we can set a context for default production value. Maybe something like this:
`<field name="context">{'default_object_id': 'mrp.production,' + str(active_id)}</field>`
or add the same context to the button that calls this action?